src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/listeners/CustodyLocationLookupListener.kt

```diff
@@ -18,7 +19,8 @@ class CustodyLocationLookupListener(
   val telemetryClient: TelemetryClient,
 ) : ApplicationListener<ReferralEvent> {
   override fun onApplicationEvent(event: ReferralEvent) {
-    if (event.type != ReferralEventType.SENT) {
+    if (event.type != ReferralEventType.SENT || event.referral.referralLocation?.type != PersonCurrentLocationType.CUSTODY
```

Comment:
Is this not `if (event.type != ReferralEventType.SENT && event.referral.referralLocation?.type != PersonCurrentLocationType.CUSTODY` ?